VAST_INTEGRATION_GUIDE.md

```
@@ -0,0 +1,443 @@
+        print("Error: \(error.localizedDescription)")
+    }
+)
+.frame(height: 220)
+```
+
+## Configuration Options
+
+### VideoAdConfiguration
+
+| Property | Type | Default | Description |
+|----------|------|---------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)